apps/wallet/src/ui/app/pages/home/bullshark-quests-notification/index.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { X32 } from '@mysten/icons';
+
+import { Portal } from '../../../shared/Portal';
+
+const InterstitialHeading = ({ text }: { text: string }) => {
+	return <div className="[-webkit-text-stroke:2px_black] text-heading1">{text}</div>;
+};
+
+const InterstitialBanner = ({ lines }: { lines: string[] }) => {
+	return (
+		<div className="text-heading4 flex flex-col border-solid border-2 border-gray-90 text-black bg-white py-1 px-3 rounded-md">
+			{lines.map((line, key) => {
+				return <div key={key}>{line}</div>;
+			})}
+		</div>
+	);
+};
+
+function BullsharkQuestsNotification({ onClose }: { onClose: () => void }) {
+	const setInterstitialDismissed = () => {
+		localStorage.setItem('bullshark-interstitial-dismissed', 'true');
+	};
+
+	return (
+		<Portal containerId="overlay-portal-container">
+			<div className="flex flex-col justify-center flex-nowrap items-center bg-[#99dbfb] border-solid border-4 border-black rounded-lg z-50 overflow-hidden absolute top-0 bottom-0 left-0 right-0 backdrop-blur-sm">
+				<div className="flex flex-col font-frankfurter w-full h-full items-center p-4 text-white text-center">
+					<div className="flex flex-col py-6 px-7 border-4 border-black border-solid w-full rounded-md h-full items-center">
+						<div className="flex flex-col items-center">
+							<InterstitialHeading text="Join Bullshark Quests!" />
+							<div className="bg-[url('https://quests.mystenlabs.com/_next/static/media/logo.81b4eb8f.svg')] h-40 w-40 bg-cover" />
+							<InterstitialHeading text="5 Million SUI prize pool!" />
+							<div className="flex flex-col items-center gap-2 mt-2">
+								<InterstitialBanner lines={['2.5M SUI', 'Top 10,000 players!']} />
+								<InterstitialBanner lines={['2.5M SUI', 'all eligible players!']} />
+							</div>
+						</div>
+
+						<div className="flex flex-col items-center gap-4 [-webkit-text-stroke:1px_black] w-full mt-5">
+							<a
```

Comment:
Do we need `target="_blank"` for these a tags? I see an `ExternalLink` component, and wonder if we should be using that instead?